### PR TITLE
Feat/asset system

### PR DESCRIPTION
**Pull request by Vafdaf12**

### CODE REVIEW

File: assets/hunny.png
Comment (21434809) [inline assets/hunny.png:1]: HAHahahahHAHAhAHHAh!!!!!!

File: assets/textures/hunny.png
Comment (21434809) [inline assets/textures/hunny.png:1]: HJAHAHAH XD!

Comment (Vafdaf12) [reply assets/textures/hunny.png:1]: poggers

**Review by 21434809 — APPROVED** [review]
awes

**Pull request merged**